host/3.0/buster/amd64/dotnet/dotnet7-isolated/dotnet7-isolate-composite.template

```diff
@@ -0,0 +1,35 @@
+FROM mcr.microsoft.com/dotnet/core/runtime-deps:3.1
```

Comment:
Thanks @CooperLink  for the sample !

@kshyju - please feel free to use this PR as a starting point and update to install .Net 7